python/ray/tune/syncer.py

```diff
@@ -209,6 +212,8 @@ def reset(self):
         self.sync_client.reset()
 
     def close(self):
+        # Force one last sync before closing out.
```

Comment:
This is sort of "best effort". If any file content is updated on worker after client is closed, there is much we can do anyways.